tracer/src/Datadog.Trace/Propagators/SpanContextPropagator.cs

```diff
@@ -49,9 +49,14 @@ public static SpanContextPropagator Instance
                         return _instance;
                     }
 
-                    var distributedContextPropagator = (IContextExtractor)new DistributedContextExtractor();
-                    var datadogPropagator = new DatadogContextPropagator();
-                    _instance ??= new SpanContextPropagator(new[] { datadogPropagator }, new[] { distributedContextPropagator, datadogPropagator });
```

Comment:
Seems surprising we weren't using the static instances before? 🤔 These are stateless right, so there would be no reason not to? Also AFAICT I assume `_instance` can never be _set_ to `null` once it's non-null (hence dropping `??=`)?

Looks right to me, just confirming I'm not missing something! 😅

Reply:
Yes, they are stateless, but I only added the static instances recently. I thought I changed all usages when I did that, but I must have missed this one. (I should probably add private ctors, too, in separate PR).

You're correct that `_instance` can never be _set_ to `null`. But also we already check if it is `null` (the first time) a few lines above.